src/Hooks.php

```diff
@@ -146,62 +122,31 @@ public static function onUserGroupsChanged( $user, array $add, array $remove, ..
 
 	/**
 	 * Add sysop to effective groups when the user has wiki_guardian.
-	 *
-	 * @param mixed &$user User
-	 * @param array &$aUserGroups "Actual" user groups that should reflect the rows in the database.
-	 *
-	 * @return void
+	 * @inheritDoc
 	 */
-	public static function onUserEffectiveGroups( &$user, &$aUserGroups ): void {
-		if ( in_array( 'wiki_guardian', $aUserGroups ) ) {
-			$aUserGroups[] = 'sysop';
+	public function onUserEffectiveGroups( $user, &$groups ) {
+		if ( in_array( 'wiki_guardian', $groups ) ) {
+			$groups[] = 'sysop';
 		}
 	}
 
-	/**
-	 * Setups and Modifies Database Information
-	 *
-	 * @param mixed|null $updater [Optional] DatabaseUpdater Object
-	 *
-	 * @return bool true
-	 */
-	public static function onLoadExtensionSchemaUpdates( DatabaseUpdater $updater = null ) {
+	/** @inheritDoc */
+	public function onLoadExtensionSchemaUpdates( $updater ) {
 		$extDir = __DIR__ . '/../';
 
 		// Tables
 		// 2015-01-08
-		$updater->addExtensionUpdate( [
-			'addTable',
+		$updater->addExtensionTable(
 			'wiki_claims',
-			"{$extDir}/install/sql/claimwiki_table_wiki_claims.sql",
-			true,
-		] );
-		$updater->addExtensionUpdate( [
-			'addTable',
+			"$extDir/install/sql/claimwiki_table_wiki_claims.sql"
+		);
+		$updater->addExtensionTable(
 			'wiki_claims_answers',
-			"{$extDir}/install/sql/claimwiki_table_wiki_claims_answers.sql",
-			true,
-		] );
-		$updater->addExtensionUpdate( [
-			'addTable',
+			"$extDir/install/sql/claimwiki_table_wiki_claims_answers.sql"
+		);
+		$updater->addExtensionTable(
 			'wiki_claims_log',
-			"{$extDir}/install/sql/claimwiki_table_wiki_claims_log.sql",
-			true,
-		] );
-
-		return true;
-	}
-
-	/**
-	 * Setup Twig Templates
-	 *
-	 * @param SpecialPage $special
-	 * @param string $subPage
-	 *
-	 * @return void
-	 */
-	public static function onSpecialPageBeforeExecute( SpecialPage $special, $subPage ) {
-		$twig = MediaWikiServices::getInstance()->getService( 'TwiggyService' );
-		$twig->setTemplateLocation( 'ClaimWiki', __DIR__ . '/../resources/templates' );
```

Comment:
Removed to avoid service pollution. Each class now handles TwiggyService configuration